example/cumulus-tf/cnm_response_task.tf

```diff
@@ -1,7 +1,7 @@
 resource "aws_lambda_function" "cnm_response_task" {
```

Comment:
These functions will additionally need to be updated with the CMA lambda layer, and probably require some more fixes before `KinesisTestTriggerSpec` will pass. cc @Jkovarik